### PR TITLE
Function annotations

### PR DESCRIPTION
**Pull request by kevinoliveira**
**sdkgen function annotations are now displayed on the annotations tab** 
_code annotations_
code annotations
_playground annotations tab_
playground annotations tab

 **Other things that are not related to this feature**
- few packages were updated because of potential vulnerabilities.
- previous updates on the icon library have broken the types, use of  `//@ts-ignore` was made.
- the endpoint URL on development mode is now `http://localhost:8000`, the port can be changed at will using the `SERVER_PORT` environment variable.

### CODE REVIEW

File: src/components/requestCard/annotations/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+interface Props {
+	annotations: ModelAnotations;
+}
+
+export function Annotations(props: Props) {
+	const { args, func } = props.annotations;
+
+	const isEmpty = func.length === 0 && Object.keys(args).length === 0;
+
+	const emptyMessage = <div className={s.empty}>There is no annotations on this function.</div>;
```
Comment (lbguilherme) [inline src/components/requestCard/annotations/index.tsx]: Typo: There *are* no annotations

File: src/components/requestCard/annotations/index.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+				<div className={s.name}>{argName}</div>
+				<div className={s.content}>{groupedAnnotations}</div>
+			</div>,
+		];
+	}, []);
+
+	const argsSection =
+		argsAnnotations.length !== 0 ? (
+			<div className={s.section}>
+				<div className={s.title}>Arguments annotations</div>
```
Comment (lbguilherme) [inline src/components/requestCard/annotations/index.tsx]: Just "Arguments" looks better

File: src/components/requestCard/annotations/index.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+		<div className={s.annotation} key={`func-${index}`}>
+			<div className={s.type}>{a.type}:</div>
+			<div className={s.value}>{a.value}</div>
+		</div>
+	));
+
+	const funcSection =
+		funcAnnotations.length !== 0 ? (
+			<div className={s.section}>
+				<div className={s.title}>Function annotations</div>
```
Comment (lbguilherme) [inline src/components/requestCard/annotations/index.tsx]: Just "Function" looks better

File: src/components/requestCard/annotations/index.tsx (hunk 1 above)
Comment (kevinoliveira) [reply src/components/requestCard/annotations/index.tsx]:
```suggestion
	const emptyMessage = <div className={s.empty}>There are no annotations on this function.</div>;
```

File: src/components/requestCard/annotations/index.tsx (hunk 2 above)
Comment (kevinoliveira) [reply src/components/requestCard/annotations/index.tsx]:
```suggestion
				<div className={s.title}>Arguments</div>
```

File: src/components/requestCard/annotations/index.tsx (hunk 3 above)
Comment (kevinoliveira) [reply src/components/requestCard/annotations/index.tsx]:
```suggestion
				<div className={s.title}>Function</div>
```

**Comment by kevinoliveira** [comment]
@lbguilherme changes made.

**Pull request merged**